### PR TITLE
Added header

### PR DESCRIPTION
**Pull request by MDGcode**

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -1,64 +1,15 @@
...
   margin: 0 auto;
   padding: 2rem;
   text-align: center;
+}}
+#root {
+  /* max-width: 1280px; */
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: center;
 }
```
Comment (cristim67) [inline client/src/App.css:15]: format this.

File: client/src/components/HeaderMobile.Component.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        <div className="  bg-black">
+          <AiOutlineMenu
+            onClick={handleNav}
+            className="fixed top-8 right-4 z-[99] md:hidden text-blue-950 outline-4 font-extrabold"
+            size={20}
+            color="white"
+          />
+        </div>
+        {content}
+      </div> */}
```
Comment (AMihneaa) [inline client/src/components/HeaderMobile.Component.tsx:117]: Delete this

**Review by AMihneaa — APPROVED** [review]

File: client/src/App.css (hunk 1 above)
Comment (AMihneaa) [reply client/src/App.css:15]: agree :D

File: client/src/components/HeaderMobile.Component.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+          className="border-b-2 pb-2 w-1/2 border-[#ffc947]"
+        >
+          <button>
+            <p className="text-white/90 text-frumos text-nowrap pr-10 text-left">
+              Program
+            </p>
+          </button>
+        </a>
+        <a
+          onClick={handleNav}
```
Comment (cristim67) [inline client/src/components/HeaderMobile.Component.tsx:70]: move onClick in button TE ROG EU

File: client/src/components/HeaderMobile.Component.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
+import { useState } from "react";
+import { AiOutlineMenu } from "react-icons/ai";
+import { IoMdClose } from "react-icons/io";
+import Logo from "../assets/Logo2.svg";
+export default function HeaderMobile() {
+  const [nav, setNav] = useState(false);
+  const handleNav = () => {
+    setNav(!nav);
```
Comment (cristim67) [inline client/src/components/HeaderMobile.Component.tsx:8]: If the navigation links are meant to scroll to sections on the page, ensure you prevent the default anchor behavior (e.preventDefault()) in your handleNav function.

Comment (MDGcode) [reply client/src/components/HeaderMobile.Component.tsx:8]: Thx

Comment (AMihneaa) [reply client/src/components/HeaderMobile.Component.tsx:8]: const handleNav = (ev: eventType) => {
   ev.preventDefault();
   setNav(!nav)
}

**Pull request merged**